packages/integration-tests/fixtures/disabled-debug-id-injection/disabled-debug-id-injection.test.ts

```diff
@@ -6,10 +6,10 @@ import { testIfNodeMajorVersionIsLessThan18 } from "../../utils/testIf";
 
 function checkBundle(bundlePath1: string, bundlePath2: string) {
   const process1Output = childProcess.execSync(`node ${bundlePath1}`, { encoding: "utf-8" });
-  expect(process1Output).toBe("undefined\n");
```

Comment:
Looks like newer versions of Node don't have the newline at the end of the output because these were failing on v20 and v22!